src/main/java/com/ericsson/eiffel/remrem/shared/MsgService.java

```diff
@@ -4,5 +4,12 @@
 import com.google.gson.JsonObject;
 
 public interface MsgService {
+    
+    String EVENT_ID = "eventId";
```

Comment:
This constants will use by both remrem-semantics and eiffel3messaging.

**The added getEventId method implementation :** 
We did not convert the json object to java objects. From json object only we trying to get event id. in this process we need some constants so i placed here.